docs/config/README.md

```diff
@@ -1465,13 +1465,22 @@ The module will be shown if any of the following conditions are met:
 - The current directory contains a `.ruby-version` file
 - The current directory contains a `.rb` file
 
-### Options
```

Comment:
I think it is worth keeping this line to keep this inline with the other modules.

Reply:
@andytom Whoops didn't even notice this 😅 Added it back :)